package.json

```diff
@@ -11,8 +11,8 @@
   },
   "dependencies": {
     "@vweevers/docker-pull": "^1.1.1",
+    "@thegecko/docker-run": "^3.0.0",
```

Comment:
The last version from [mafintosh was 3.0.0](https://github.com/mafintosh/docker-run) which was where this was forked from. The latest version on [npm is 3.1.0](https://www.npmjs.com/package/docker-run) published by [maxogden](https://github.com/max-mapper) however I can't seem to locate the source for this. Who knows what features this is removing :/

Reply:
Oh, that's odd. But the diff is minimal, so you can just copy a few lines: https://socket.dev/npm/package/docker-run/diff/3.1.0

Reply:
https://github.com/thegecko/docker-run/pull/2

Reply:
Updated